content/tr/application-programming-interface.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Uygulama Programlama Arayüzü (API)
+status: Tamamlandı
+category: Teknoloji
+tags: ["mimari", "temel", ""]
```

Comment:
`
status: Completed
category: Teknoloji
tags: ["mimari", "temel kavram", ""]
`
Can you go like above?

We didn't translate `Completed` in other pages. It is not visible to users anyway.

Let's update `tags` to be consistent with other pages.

Reply:
Okay. That's going to be edited in a few seconds.

Reply:
#2703